tests/scripts/all.sh

```diff
@@ -2689,7 +2688,8 @@ common_test_psa_crypto_config_accel_ecc_some_curves () {
     # -------------
 
     msg "test suites: full with accelerated EC algs and $DESC curves"
-    make test
+    # does not work for PK (and above), see #8255
+    make test SKIP_TEST_SUITES=pk,pkparse,pkwrite,x509parse,x509write,ssl,debug
```

Comment:
I'm willing to approve this as an emergency for the release if we really feel that we can't afford the time of one more CI run, but I'd prefer to set up a configuration without `MBEDTLS_PK_C` or X.509/TLS (i.e. start with `config.py crypo_full; config.py unset MBEDTLS_PK_C` instead of `config.py full`) and then run a plain `make test`.

Reply:
Why would you prefer removing things are compile time? Currently everything builds, so why not assert that so that it keeps building in the future?

Reply:
Note: in general, I'd really like to merge this ASAP, in order to unblock #8242. That doesn't prevent us from getting a follow-up PR in the release with the subset of fixes we care about (and possibly more depending on how other release-critical things are progressing).

Reply:
> Why would you prefer removing things are compile time?

Because “it builds but we don't know what it does” is worse than “it's unsupported, don't do this”. I'd even prefer to have an `#error` than mysterious errors that may happen either at compile time or run time.

Reply:
It sounds like we should keep this PR as-is, to make progress, but in parallel prepare a PR to add a test as Gilles suggests?

Reply:
>Because “it builds but we don't know what it does” is worse than “it's unsupported, don't do this”.

The documentation doesn't say it builds, it says "there are known issues in PK, and X.509 and TLS are untested" which I think is much closer to "it's unsupported, don't do this".

> I'd even prefer to have an #error than mysterious errors that may happen either at compile time or run time.

Now, that's an idea I like: not everyone reads the documentation, so an `#error` is safer. But then it goes beyond your initial comment. I can see the point of not building if we take advantage of it in order to give an `#error`.